libs/sdk-react-native/src/index.tsx

```diff
@@ -515,6 +534,14 @@ export const connect = async (config: Config, seed: Uint8Array): Promise<void> =
     await BreezSDK.connect(config, seed)
 }
 
+export const signMessage = async (request: SignMessageRequest): Promise<SignMessageResponse> => {
```

Comment:
I'm not clear whether the signMessage / checkMessage functions should take string arguments for the Request objects.